kura/org.eclipse.kura.core.keystore/OSGI-INF/metatype/org.eclipse.kura.core.keystore.FilesystemKeystoreServiceImpl.xml

```diff
@@ -45,8 +45,88 @@
             description="Specifies if the defined password will be randomized at the next keystore access. If this value is set to true and the keystore can be accessed, the password will be randomized and this field will automatically set to false.">
         </AD>
 
+        <AD id="crl.management.enabled"
```

Comment:
Maybe we need to expand the description. As a user I'm not sure what this option does